cmd/dptp-controller-manager/main.go

```diff
@@ -79,6 +81,17 @@ type testImagesDistributorOptions struct {
 	forbiddenRegistries                sets.String
 }
 
+type registrySyncerOptions struct {
+	excludeImageStreamTagsRaw       flagutil.Strings
+	excludeImageStreamTags          sets.String
+	excludeImageStreamsRaw          flagutil.Strings
+	excludeImageStreams             sets.String
+	excludeImageStreamNamespacesRaw flagutil.Strings
```

Comment:
I don't think excludedNamespaces or forbiddenRegistries make a lot of sense for this controller